### DISCUSSION TITLE
How do i setup Board for diffrent panel?

### DISCUSSION THREAD

**Discussion by superuser-me** [mokhosh/filament-kanban#43] [Q&A] [answered]
I have two panels in my project.
By default it's showing up in my admin (default) panel.
how do i change this and register this for different panel,
I am on filament version 3.2.

I am new to laravel /filament development, sorry for this basic rookie question.

**Comment by mokhosh** [comment, marked as answer]
In your panel provdier you have a `discoverPages` method which shows the folder where the pages for that panel should be. Let's say `App\Filament\Student\Pages` and your panel is called `student`.
You would move the kanban board from `App\Filament\Pages` to `App\Filament\Student\Pages` and it will appear in the `student` panel.
